ui/src/settings.ts

```diff
@@ -31,6 +33,22 @@ interface Option<T> {
   title: string;
 }
 
+export enum Theme {
+  Dark = "dark",
+  Light = "light",
+  H4x0r = "h4x0r",
+}
+
+export enum UIFont {
+  Inter = "inter",
+  System = "system",
+}
+
+export enum CodeFont {
+  SourceCode = "sourceCode",
+  System = "system",
+}
```

Comment:
These definitions should be moved into `proxy/settings.ts` and you need to update the [`Appearance` schema](https://github.com/radicle-dev/radicle-upstream/blob/b8c4ad4afb781342c3446beebf57ee36e3f42b35/ui/src/proxy/settings.ts#L31-L36) to ensure we properly decode this. (Happy to help if you need support with that).

Reply:
https://github.com/radicle-dev/radicle-upstream/pull/1754/commits/1cfc90ec9a3d1ed94fabbc9431509d5efde31afc